pombola/feedback/views.py

```diff
@@ -18,6 +20,8 @@ def add(request):
     # If it is a post request try to create the feedback
     if request.method == 'POST':
         form = FeedbackForm( request.POST )
+        recaptcha_response = request.POST.get("g-recaptcha-response", "")
```

Comment:
I think a default of None is better, since that more clearly means "no value" instead of "the empty string value".
Does the recaptcha client handle None correctly?